### PR TITLE
Adds the flag to not inlude the publicPath

### PR DESCRIPTION
**Pull request by ernestorocha**
For some reason the indent changed complete, probably the `eslint --fix`.

But the key idea here is only to get the asset name, in my case, I'm going to serve then thru a CDN.

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -219,51 +219,74 @@ describe('Plugin', function () {
...
+      output: {
+          path: OUTPUT_DIR,
+          publicPath: '/public/path/[hash]/',
+          filename: 'index-bundle.js',
+          fullPath: false
+        },
+      plugins: [new Plugin({path: 'tmp'})]
+    };
+
+    var expected = new RegExp('index-bundle.js', 'i');
```
Comment (sporto) [inline test/index.test.js]: This should check for the beginning of the string:

```
var expected = new RegExp('\Aindex-bundle.js', 'i');
```

**Comment by sporto** [comment]
@ernestorocha   Thanks for the PR, could you fix the test so it checks that the string appears at the beginning of the output?

**Comment by ernestorocha** [comment]
Actually the test was wrong, since the whole output is being converted to string. Thanks for pointing that out!

But again, a lot of indention changed. If you prefer I can prevent the indention of those files to change.

**Comment by sporto** [comment]
published v3.1.0

**Comment by ernestorocha** [comment]
Thanks a lot, @sporto!

**Pull request merged**